gwpy/utils/lal.py

```diff
@@ -190,25 +182,36 @@ def to_lal_unit(aunit):
     ValueError
         if LAL doesn't understand the base units for the input
     """
+    # format incoming unit
     if isinstance(aunit, str):
         aunit = units.Unit(aunit)
     aunit = aunit.decompose()
-    lunit = lal.Unit()
+
+    # handle scaled units
```

Comment:
Minor comment: it would be helpful to explain what "handle scaled units" means in this context. Handle in what way?

Reply:
This is now explained in the docstring for the function, see 1ee6961b134323ebbd5fc122ee96dbaf9b6c34b4.